src/commands/integration-test-create-cluster.yaml

```
@@ -0,0 +1,16 @@
+parameters:
+  kind-config:
+    type: string
+steps:
+  - unless:
+      condition: << parameters.kind-config >>
+      steps:
+        - run:
+            name: "Prepare default kind config"
+            command: |
+              echo "kind: Cluster" >> << parameters.kind-config >>
+              echo "apiVersion: kind.sigs.k8s.io/v1alpha3" >> << parameters.kind-config >>
+  - run:
+      name: "Create cluster"
+      command: |
+        kind create cluster --config  << parameters.kind-config >>  --image quay.io/giantswarm/kind-node:$KUBERNETES_VERSION
```

When << parameters.kind-config >> is empty `echo "kind: Cluster" >> << parameters.kind-config >>` would write to file "".

```suggestion
  - when:
      condition: << parameters.kind-config >>
      steps:
        - run:
            name: "Move kind config"
            command: |
                cp << parameters.kind-config >> .kind-config
  - unless:
      condition: << parameters.kind-config >>
      steps:
        - run:
            name: "Prepare default kind config"
            command: |
                printf "kind: Cluster\napiVersion: kind.sigs.k8s.io/v1alpha3\n" .kind-config
  - run:
      name: "Create cluster"
      command: |
          kind create cluster --config  .kind-config  --image quay.io/giantswarm/kind-node:$KUBERNETES_VERSION
```

---

Yes I tried to remove the file copy but its needed. Going with this, thx!